src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java

```diff
@@ -0,0 +1,126 @@
+/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```

Comment:
New file, the copyright is just "(c) 2015 - Open Source Geospatial Foundation - all rights reserved"